### PR TITLE
Upgrade Benchmarker

### PR DESCRIPTION
**Pull request by GorogPeter**
- "--verbose" flag is added. If it's set, it works similarly to the older version.
- if "--verbose" flag is not set, it works more _user friendly_. Only essential informations are printed on the screen, and progress bar shows the current status.
- Result saving is upgraded. Now, test results can be saved as CSV.

### CODE REVIEW

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -230,14 +230,15 @@ jobs:
         sudo apt update
         sudo apt install -y ninja-build gcc-multilib g++-multilib
         sudo pip install py-markdown-table
+        sudo pip install tqdm
```
Comment (zherczeg) [inline .github/workflows/actions.yml:233]: Why do we need this? Could we avoid it easily?

Comment (GorogPeter) [reply .github/workflows/actions.yml:233]: It's needed for printing the progress bar in _non-verbose_ mode. With it, it's much easier to print nice progress bar. Like `sudo pip install py-markdown-table` (previous line), it's a _comfort option_. Not necessary but useful.

Is it fine or I should remove it?

Comment (zherczeg) [reply .github/workflows/actions.yml:233]: Let us wait for others opinion.

Comment (ksh8281) [reply .github/workflows/actions.yml:233]: It's fine!

**Review by ksh8281 — APPROVED** [review]

**Pull request merged**